vscode/microsoft-kiota/l10n/bundle.l10n.tr.json

```diff
@@ -23,7 +23,7 @@
   "Enter an output path relative to the root of the project": "Proje köküne göre bir çıktı yolu girin",
   "Pick a language": "Bir dil seçin",
   "Downloading kiota requires an internet connection. Please check your connection and try again.": "Kiota'nın indirilmesi bir internet bağlantısı gerektirir. Lütfen bağlantınızı kontrol edin ve tekrar deneyin.",
-  "Kiota download failed. Try closing all Visual Studio Code windows and open only one. Check the extension host logs for more information.": "Kiota indirme işlemi başarısız oldu. Tüm Visual Studio Code pencerelerini kapatmayı ve yalnızca birini açmayı deneyin. Daha fazla bilgi için extension host loglarını kontrol edin.",
+  "Kiota download failed. Try closing all Visual Studio Code windows and open only one. Check the extension host logs for more information.": "Kiota indirme işlemi başarısız oldu. Tüm Visual Studio Code pencerelerini kapatmayı ve yalnızca birini açmayı deneyin. Daha fazla bilgi için extension host loglarını kontrol edin",
```

Comment:
The only change here is the removal of the period and some whitespace changes. 
Just to confirm this is the only intended changes?

Reply:
hummpppffff I already had done it in #4395 thanks for calling this out. closing